packages/experimental/Overflow/src/Overflow/Overflow.types.ts

```diff
@@ -0,0 +1,66 @@
+import type { View, ViewProps, ViewStyle } from 'react-native';
+
+import type { LayoutRectangle } from '@office-iss/react-native-win32';
+
+import type { OverflowItemEntry, OverflowUpdatePayload } from '../overflowManager.types';
+
+export type LayoutSize = Pick<LayoutRectangle, 'width' | 'height'>;
+
+export const overflowName = 'Overflow';
+
+export interface OverflowProps extends ViewProps {
+  // Horizontal padding to add to the container
```

Comment:
I would make these [JSDoc comments](https://www.typescriptlang.org/play?filetype=js&useJavaScript=truee=4#code/PTAEBUE8AcFMGUDGAnAltALqAFgQwM6gBusykoai2oAUvACID2io+ArtNI8hgDSgAzbqFygANoyyMBoRAVj4AUCFCRGbWbgB2oWCR34A1ulABbXMa0BzVeuSDUYhaAB0GQtoAmoAFZt8WP6wtAzMymDaWpK4GKiMWoQYjLLIsDHBopTUnnqwEtCmsFpYRUSoyPGFxS7h4aAAgiFMLIiMplVYqB5mbGKxALRiqFrBre1FWADu2KhUrBi4PISTqBjYdRiTyQGLhMMBad7SoPGwNcoAVBAze91RyOZism0doBfAl1fgN6BdIk3MZ7jYpvD51OjNIEdQgAI1gYwyGAWVFg3iSoDWowk+AUWBouCIuCQaEwdVaOVAcIkk1+GHOrQSWFMkAAaotULgYU5QABeUAAIgAEqh+QBuRR1ACSMjUGmwjBI9gVpDMrPZnKc-FlmgMsGCaxitJwBAx2FgG1gAA8sAIKqZfglUBTMQCWi8JiIkbgUZ5zioIYCxtCRKl-pNcOR0dAKmVnTBgsMhA8YnEdEk6lA4MT0FgvLY2PZUatuPgXKAADKwDAAckIOx4oBWaxEOjS+HIpzqOKRwxsokJaA1sFrGPjGOSohhbEcGH6w1HcD9YAAYsJcGInsdMTjdJbcKZoE58Fr1DqcMqlYrTcEtPvYLw6nn4tfQCNreJhvqyI20BhggBtK190PWAAF0y3RHF9TNR82CSfoxhAv8TkwVNSwlYALiuAABDAxwAby0NhTDhZAAF9QUUAdVQAOWI0jxTqABNU85F1DINxOGQXXYThuD-NFcCsPY0zNY0tE8GFGEYQwAC5ahUbAkWgfBZJASYNLceN8BQHMxG0KwXG4KxgE8Zh8GAPAJKkmTgDwuAELNRBLCsfofAJAhdMwfoBEcBQXCU0wxAAYl4rgeFRNz8DMxAMLAQVGEmXJkH4JLq04jBvysZI1gqNgrGofA2mCLcxNMbhRheZ8gIPI86jNVIyxYjQ2JEMQiueaByAAMlAaACGQ7RyBqkDCFtNoXysyTpMMOphnRBqzji0BJQPfjexfeznCTWgPOzTBni0XyrALFNn18o95M+UBcII341p4AAKfkkphfrnP5ABKFwAGF4mO0AKPeRQGQCQ6Ab5fCyMYlQftSFNrH+RCnEtBd9WSVJ+iCB0zCGvr9MQBQrswi5FDeG6tpyGR8IAeRhHx4QwCiAFUcXsfp-lZlVvVaNhijJnDo0YOAeHIfCAjQawKM8LpDwjGi71ADmXVvQpQCCNFtnlGkXSCZABZuoWRbw0BCPo0gKKdJX-j51AAEc2ATTwBbBFRviKdWd1WSlyFSARSCKRANpdSnYAEEdVdgYmsIpu6ufsIGPlBwI2d5U3oeW93UhHF1MwQLysEQlMuQTLQhhGNHWHlHgpofFRplmahtVa7Gdqk5stpbNF4yp6OcM7-D8JOSYRmQWTQHj-hI-HiWNrIxOQfiMHUiKgtCZhmPbrgKnTfwxLR-HyfXzvGfMt7efQAAJQUOxCco66t+CfDr9X5BCYX5OTkxZAX9v2AYbAPQeE+lJZ9jRt4AQfNECxHiH3A29RPCeESFsV85tkCJEYFYKsDUDbYX6sgfcpsiIkQtv8d2Dh0FYGIaRXB+DCFmxIeRSk1xgg4gZN4ahpBcGpAwAWBIRC0FkRdooSBWhoGphEIg8AWw6KMPwI9XA-AYSfVNmTCgVY+H-AANSUnFEI5irFtCe2COVMGxw877RrIkMcSRGDtX4EMQwwRbZLz7rHOAptHqz0RgAH0pNJJw2hPoLycFgbxVgabIAAEIBLSDoPk-IxSKHCZEmJdi4lpwEOuHEADQAAFFrRFBlojKwEgYTZJxgGFgfxRDlVDMMIgdiSDeCFoTfASh64zDmM3IxUEcYuhZPAUAf0KQxVUgpMASkMAqTUsAckZwyjsGybwmWjAXBjFMuZYAUQcg+AslsZALl+hNmwG5DyOkSQYGCgAfVKYwcpYh+gDg5CXfA-QvD9C2o5eELlloIOKTYKpUIJgYPzPYUR4il6-G6CsLQxzhhdlWGwM6WhRT5lAFgrAcIkQqlseXPsElWDJDMm1J4sp9YqHqAABUlIddghR0EuCAA) so developers can read the comment when they hover over the prop in their IDE. Most of our other components do this as well. Example:

```
/**
 * Horizontal padding to add to the container
 */